docs/workflows/best-practices-organization.md

```diff
@@ -8,12 +8,13 @@ The purpose of this document is to provide concrete and actionable guidelines on
 
 ## Three stakeholder groups in a hybrid model
 
-Typically, nobody in an organization is opposed to building quality software. Rather, investing in software quality often loses out to other topics, typically adding features and fixing bugs. The shift that needs to take place is that organizations allow for a fair competition to take place between feature work and technical improvements. 
+Typically, nobody in an organization is opposed to building quality software. Rather, investing in software quality often loses out to other topics, typically adding features and fixing bugs. The shift that needs to take place is that organization arrive at a way of working that allows for a fair competition to take place between feature work and technical improvements. 
 
 Having consulted with many organizations, we have identified three stakeholder groups that need to be involved to make this shift and achieve high quality software, structurally. These are:
 - Teams
 - Engineering leads
-- Management
+- The User Organization
```

Comment:
Analogous to the other group names, perhaps just "Users" instead of "User Organization"?

Reply:
@ShimaRafS

Reply:
I upgraded 'management' to a proper stakeholder